addons/mail/static/src/components/thread_view/thread_view.xml

```diff
@@ -38,7 +38,7 @@
                         isDoFocus="props.isDoFocus"
                         showAttachmentsExtensions="props.showComposerAttachmentsExtensions"
                         showAttachmentsFilenames="props.showComposerAttachmentsFilenames"
-                        textInputSendShortcuts="(threadView.thread.model === 'mail.channel' and threadView.thread.mass_mailing) ? ['ctrl-enter', 'meta-enter'] : ['enter']"
+                        textInputSendShortcuts="(threadView.thread.model === 'mail.channel' and threadView.thread.mass_mailing) ? ['ctrl-enter', 'meta-enter'] : ['ctrl-enter', 'enter']"
```

Comment:
Should only be if isMobile is true. I don't know why (you'll need to investigate) but when it's a small screen, the "enter" here is ignored and makes a new line instead. I also posted this on the task.